usr.bin/at/at.c

```diff
@@ -880,8 +880,9 @@ main(int argc, char **argv)
 
 	if (atverify)
 	{
-	    struct tm *tm = localtime(&timer);
-	    fprintf(stderr, "%s\n", asctime(tm));
+	    char time_string[26];
+	    strftime(time_string, sizeof(time_string), "Www Mmm dd hh:mm:ss yyyy\n", localtime(&timer));
```

Comment:
This makes no sense. Did you test these changes? That's not a valid format string.